ui/app/src/components/RecentActivityDashlet/RecentActivityDashlet.tsx

```diff
@@ -91,24 +91,29 @@ export default function RecentActivityDashlet() {
   const { itemsByPath, isFetching } = useDetailedItems(Object.keys(selectedLookup));
 
   const isAllChecked = useMemo(
-    () => !items.some((item) => !item.stateMap.deleted && !selectedLookup[item.path]),
+    () =>
+      items.length > 1
+        ? !items.some((item) => !item.stateMap.deleted && !selectedLookup[item.path])
+        : items[0]
+        ? selectedLookup[items[0].path]
+        : false,
```

Comment:
What if the single item that's present is also deleted? Out of the `items.some(...)`, there's no validation of `.deleted`. How come?